pkgs/development/tools/misc/tcptrack/default.nix

```
@@ -1,16 +1,26 @@
-{ lib, stdenv, fetchFromGitHub, ncurses, libpcap }:
+{ lib, stdenv, fetchFromGitHub, fetchpatch, ncurses, libpcap }:
 
 stdenv.mkDerivation rec {
   pname = "tcptrack";
-  version = "1.4.3";
+  version = "1.4.3+unstable=2017-04-29";
```

We don't have a standard written in stone yet, but this seems to be a popular choice.

```suggestion
  version = "unstable-2017-04-29";
```

---

also see https://github.com/NixOS/rfcs/pull/107/files#diff-4fd75917011dd2a385c5914029adc59f042e5c83c5dc922f37f73891c1152dd7R27